pkg/ipam/crd.go

```diff
@@ -623,5 +623,10 @@ func (a *crdAllocator) Dump() (map[string]string, string) {
 
 // RestoreFinished marks the status of restoration as done
 func (a *crdAllocator) RestoreFinished() {
-	close(a.store.restoreFinished)
```

Comment:
please use a [`Once.Do`](https://golang.org/pkg/sync/#Once.Do) here, it will guarantee that this call is only executed once. (The Once will be a field of `nodeStore`.

Reply:
thanks for the suggestion, I will try to do some modification.